tools/vscode/README.md

```diff
@@ -18,6 +18,29 @@ correctly.
 Note that it is recommended to disable VSCode Microsoft C/C++ extension and use `vscode-clangd` instead for
 C/C++ code completion.
 
+For Chinese developers, you need to set `http_proxy` `https_proxy` `all_proxy` in advance.
+
+### Tips for proxmox users
+
+If you see the following error message:
+```
+-- Performing Test HAVE_SSSE3
+-- Performing Test HAVE_SSSE3 - Failed
+-- Performing Test HAVE_AVX2
+-- Performing Test HAVE_AVX2 - Failed
+-- Performing Test HAVE_AVX512
+-- Performing Test HAVE_AVX512 - Failed
+-- Building without AVX2 support
+-- Building without AVX512 support
+CMake Error at cmake/arch.cmake:79 (message):
+  A minimum of SSSE3 compiler support is required
+```
```

Comment:
Can you clarify where this is coming from? Scroll log up and you'll see which CMake is generating this.

Reply:
I don't save the log, seems to be hyperscan